src/server.rs

```diff
@@ -47,6 +50,7 @@ impl Server {
     pub fn new_leader<W: Write + Send + 'static>(
         bank: Bank,
         entry_height: u64,
+        window_entries: Option<Vec<Entry>>,
```

Comment:
How about passing in a `Window`? We could always wrap these functions with things like `new_default_leader` to make it easier on tests (and the drone).

Reply:
can do, but still need tail -WINDOW_SIZE from the entries iterator...

also, means passing a blob_recycler down to new_leader, because that's the one broadcaster uses...